### DISCUSSION TITLE
리팩터링 3주차 안건

### DISCUSSION THREAD

**Discussion by punchdrunkard** [prgrms-web-devcourse/Team-BackFro-ArtZip-FE#268] [02.Refactoring]
.

**Comment by punchdrunkard** [comment]
![image](https://user-images.githubusercontent.com/74234333/189316374-96222261-5672-4923-8d04-27dba10d5931.png)
전시회 선택 시, 전시회 제목에도 반영되었으면 좋겠습니다!

**Reply by gitul0515** [reply]
반영했습니다. 좋은 피드백 감사합니다 👍
https://github.com/prgrms-web-devcourse/Team-BackFro-ArtZip-FE/pull/275/commits/65589ee7c06413d3bf05877dfcf614101e2ad0c5

**Comment by gitul0515** [comment]
ExhibitionCard, ReviewCard props 논의

**Comment by gitul0515** [comment]
환경변수를 사용해도 네트워크 탭에서는 API END_POINT가 여전히 노출됨 
![image](https://user-images.githubusercontent.com/80658269/189475115-2a23bd3a-7c68-4a23-9776-d4cc643790f1.png)

Next.js의 rewrites 설정을 사용하면 이를 숨길 수 있음 

![image](https://user-images.githubusercontent.com/80658269/189475136-c71fe740-1d3c-473d-befe-419d1c184681.png)

보안 측면에서 필요하다고 생각함.
다만, 도중에 이슈가 발생할 가능성이 있음. 
추후 작업해 보겠음.

**Reply by gitul0515** [reply]
혜삐: "백엔드에서 CORS 설정을 하므로 보완할 수 있을 것"

**Comment by gitul0515** [comment]
소셜 로그인 링크에서 API_ENTRY_POINT가 노출되고 있습니다. 
환경 변수를 사용해주세요.
 
![image](https://user-images.githubusercontent.com/80658269/189476069-218164b7-0b93-441b-93dc-99e6c9c434a4.png)
